multigeiger/multigeiger.ino

```diff
@@ -69,6 +70,7 @@ void setup() {
   setup_webconf(isLoraBoard);
   setup_speaker(playSound, ledTick && switches.led_on, speakerTick && switches.speaker_on);
   setup_transmission(VERSION_STR, ssid, isLoraBoard);
+  setup_ble(ssid, sendToBle);
```

Comment:
should ` && switches.ble_on` be added here?

Reply:
done